File: app/models/topic.rb
class Topic < ApplicationRecord
    belongs_to :creator, class_name: "User", foreign_key: "user_id"
    has_many :chats, dependent: :destroy
    has_many :users, through: :chats
    # has_many :creators, through: :chats, class_name: "User", foreign_key: "user_id"

    validates :name, :description, presence: true
end

File: client/src/pages/Topic.js
function Topic({ name, description }) {

    return (
        <div className="topic-div">
            <div className="box">
                <h3 onClick={() => console.log("CLICKED")}>{ name }</h3>
                <p>{ description }</p>
            </div>
        </div>
    )
}

export default Topic;